biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import './simulate-page.css';
+import { Client } from '../../client';
+import { BiogridSimulationResults } from '../../build';
+
+export const SimulatePage = () => {
+  const [simulationResults, setSimulationResults] = useState<
+    BiogridSimulationResults
+  >();
+  const [simulationRan, setSimulationRan] = useState(false);
```

Comment:
nit: could you use variable names like. ```simulationStatus``` or ```gridStatus``` for ```simulationRan```. This enables easy readability and relates to the project. You can extrapolate the function from the variable.

Reply:
Changed to an enum!